packages/styles/_utils.scss

```diff
@@ -110,8 +149,39 @@ $sides: (top, right, bottom, left);
   text-overflow: ellipsis;
   overflow: hidden;
 }
+.truncate-multi {
+  line-height: var(--TLineHeight, 24px);
+  font-size: var(--TFontSize, 16px);
+  position: relative;
+  max-height: calc(var(--TLineHeight, 24px) * var(--TMaxLines, 5));
+  overflow: hidden;
+  padding-right: calc(var(--TPosRight, 12px) * var(--TPadRight, 8px) + 4px) !important;
 
-.cursor-pointer	{ cursor: pointer !important; }
+  .truncate-multi::before {
+    position: absolute;
+    content: "...";
+    top: calc(var(--TLineHeight, 24px) * (var(--TMaxLines, 5) - 1) + var(--TFontSize, 16px) * .5);
+    right: var(--TPosRight, 12px);
+  }
+}
+.capitalize {
+  text-transform: capitalize !important;
+}
+.lowercase {
+  text-transform: lowercase !important;
+}
+.uppercase {
+  text-transform: uppercase !important;
+}
+.bold-500 {
+  font-weight: 500 !important;
+}
+.bold-600 {
+  font-weight: 600 !important;
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought:** ‏The Roboto that we use doesn't have a 600 weight. I'm not sure which weight would get rendered in the case that this was used. Possibly confusing. https://fonts.google.com/specimen/Roboto

Reply:
I don't think it matters? It's like someone trying to use `font-weight` `700` when only up to `600` is supported. It ends up rendering the highest it supports?

`font-weight` `500` exists

![image](https://user-images.githubusercontent.com/67973710/123986340-399e6080-d994-11eb-9664-420debf2add8.png)


`font-weight` `600` exists

![image](https://user-images.githubusercontent.com/67973710/123986416-4622b900-d994-11eb-8864-34fb6405681f.png)


`font-weight` `700` not supported and looks the same as `600`

![image](https://user-images.githubusercontent.com/67973710/123986371-3efbab00-d994-11eb-8c9e-548d6f0a608b.png)